deployment_scripts/deploy_linux.sh

```diff
@@ -78,87 +110,109 @@ if [[ ${python_cmd} == "" ]]; then
   log_message "Python 3.7 command not found. Installing python 3.7."
   sudo add-apt-repository ppa:deadsnakes/ppa
   sudo apt-get update
-  sudo apt install python3.7
+  sudo apt install python3.7 python3.7-dev
   log_message "Python 3.7 is now available with command 'python3.7'."
   python_cmd="python3.7"
 fi
 
-log_message "Updating package list"
-sudo apt-get update
-
-log_message "Installing pip"
-sudo apt install python3-pip
-${python_cmd} -m pip install pip
+log_message "Installing build-essentials"
+sudo apt install build-essentials
 
-log_message "Install python3.7-dev"
-sudo apt-get install python3.7-dev
+log_message "Installing or updating pip"
+# shellcheck disable=SC2086
+if exists wget; then
+  wget --output-document=get-pip.py https://bootstrap.pypa.io/get-pip.py
+else
+  curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+fi
+${python_cmd} get-pip.py
+rm get-pip.py
 
 log_message "Installing island requirements"
```

Comment:
Do we want to `pip install virtualenv` and create a `venv` instead of polluting the system python?

Reply:
In theory yes, in practice, I don't want to change any functionality in this PR beyond just having a single command for install.